ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -72,6 +73,13 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
    */
   beforeModel() {
     if (!this.session.isAuthenticated) this.router.transitionTo('index');
+
+    // Unload all sessions from the store as we might have sessions still in store
+    // if they were canceled outside of the desktop client
+    // Not wrapping it an ember run seems to cause a test to fail as well as an error
+    // if you switch between orgs due to a similar issue as this
+    // https://github.com/emberjs/data/issues/5447#issuecomment-845672812
+    run(() => this.store.unloadAll('session'));
```

Comment:
Interestingly enough, this looks like the same error as https://github.com/hashicorp/boundary-ui/pull/2267 but this hack doesn't fix that issue.

It seems the fundamental issue is some weird behavior with ember's `unloadAll` (possibly in combination with `peekAll`).